Math/Armstrong.py

```diff
@@ -0,0 +1,9 @@
+x = input()
+sum = 0
+for i in x:
+    z = int(i)
+    sum += z**3
```

Comment:
I believe Armstrong numbers are the numbers whose sum of each integer is raised to the power length of the number is the same as the number.
eg: 
- 370 = 3^3 + 7^3 + 0^3
- 1634 = 1^4 + 6^4 + 3^4 + 4^4

Reply:
yes, you are right. Sorry, for the inconvenience. I have done the changes.